models/actions/run_list.go

```diff
@@ -69,9 +69,10 @@ type FindRunOptions struct {
 	RepoID           int64
 	OwnerID          int64
 	WorkflowFileName string
+	Ref              string
```

Comment:
Should we have an index for `ref`?

Reply:
[f066eb0](https://github.com/go-gitea/gitea/pull/25716/commits/f066eb06d3c4624c9f26d1bc428b9f15649ea229) DONE